Client/Frontend/Browser/TabDisplayManager.swift

```diff
@@ -304,25 +314,20 @@ class TabDisplayManager: NSObject, FeatureFlaggable {
         guard isPrivate != isOn else { return }
 
         isPrivate = isOn
+
         UserDefaults.standard.set(isPrivate, forKey: "wasLastSessionPrivate")
 
         TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .privateBrowsingButton, extras: ["is-private": isOn.description] )
 
-        refreshStore()
-
-        if createTabOnEmptyPrivateMode {
-            // if private tabs is empty and we are transitioning to it add a tab
-            if tabManager.privateTabs.isEmpty && isPrivate {
-                tabManager.addTab(isPrivate: true)
-            }
-        }
-
-        getTabsAndUpdateInactiveState { tabGroup, tabsToDisplay in
-            let tab = mostRecentTab(inTabs: tabsToDisplay) ?? tabsToDisplay.last
-            if let tab = tab {
-                self.tabManager.selectTab(tab)
-            }
-        }
+//        if !isPrivate, let mostRecentNormalTab = mostRecentTab(inTabs: tabManager.normalTabs) {
```

Comment:
I think should we include a TODO or a ticket # to clean up or improve the commented code